data/remotes.d/lvfs.conf

```diff
@@ -7,4 +7,5 @@ Keyring=gpg
 MetadataURI=https://cdn.fwupd.org/downloads/firmware.xml.gz
 ReportURI=https://fwupd.org/lvfs/firmware/report
 OrderBefore=fwupd
+AutomaticReporting=false
```

Comment:
AutomaticReporting *and* automatic_reports? I think we want one or the other pls.